src/main/java/com/sse/ooseproject/models/Chair.java

```diff
@@ -0,0 +1,61 @@
+package com.sse.ooseproject.models;
+
+import jakarta.persistence.*;
+
+@Entity
+@Table(name = "chair")
+public class Chair extends OrganizationalUnit {
+
+    @OneToOne
+    @JoinColumn(name = "owner_id")
+    private Employee chairOwner;
+
+    @ManyToOne
+    @JoinColumn(name = "building_id")
+    private Building building;
+
+    @ManyToOne
+    @JoinColumn(name = "institute_id")
+    private Institute institute;
+
```

Comment:
Wenn ich das richtig in der datenbank sehe hat die tabelle chair auch eine university_id. Wahrscheinlich sollten wir noch ein attribute dafür hinzufügen und bei der University dementsprechend eine list of chairs.

Reply:
Dann müssten wir das aber auch bei allen anderen Klassen noch so machen. Weil es gibt auch in anderen Tabellen noch Spalten, die noch nicht als Attribut in der Klasse vorhanden sind. Weil bisher hatten wir ja immer nur die Variablen erstellt, welche auch auf dem Blatt irgendwie standen..

Reply:
Ich habe Alex mal ne Discord Nachricht geschickt